api/src/main/java/org/pmiops/workbench/db/model/DbUserTermsOfService.java

```diff
@@ -0,0 +1,75 @@
+package org.pmiops.workbench.db.model;
+
+import java.sql.Timestamp;
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import org.hibernate.annotations.CreationTimestamp;
+
+@Entity
+@Table(name = "user_terms_of_service")
+public class DbUserTermsOfService {
```

Comment:
aside: how hard would it be to merge the data model and services, etc, for the TOS and DUA. I think only a little business logic separates them, right?